sdk/enoki/src/EnokiFlow.ts

```diff
@@ -336,19 +334,51 @@ export class EnokiFlow {
 			},
 		});
 
-		const { bytes, signature: sponsorSignature } =
-			await this.#enokiClient.createSponsoredTransactionBlock({
-				jwt: session!.jwt!,
-				network,
-				transactionBlockKindBytes: toB64(transactionBlockKindBytes),
-			});
+		return await this.#enokiClient.createSponsoredTransactionBlock({
+			jwt: session!.jwt!,
```

Comment:
are the session and jwt actually guaranteed to be defined here?

Reply:
No, I'll update to handle this